package/MDAnalysis/core/universe.py

```diff
@@ -72,7 +72,7 @@
 import os
 import uuid
 
-from .. import _TOPOLOGY_ATTRS, _PARSERS
+from .. import _TOPOLOGY_ATTRS, _PARSERS, _GUESSERS
```

Comment:
I don't think you've added _GUESSERS to `__init__.py` yet.

Reply:
oh my fault! I just forgot to remove the import when I eliminated the registration. It is supposed to exist in the next pull request as I will add the guesser metaclass ,